packages/atomic/src/components/facets/atomic-color-facet/atomic-color-facet.tsx

```diff
@@ -60,6 +60,7 @@ import {Hidden} from '../../common/hidden';
  * @part values - The facet values container.
  * @part value-label - The facet value label, common for all displays.
  * @part value-count - The facet value count, common for all displays.
+ * @part value-* - The dynamic part name used to customize a facet value. The `*` is a syntactical placeholder for a specific facet value. For example, if the component's `field` property is set to 'filetype' and your source has a `YouTubeVideo` file type, the part would be targeted like this: `atomic-color-facet::part(value-YouTubeVideo)...`.
```

Comment:
I need feedback on this description; it's a bit tricky!

Reply:
I think that's a fine description 🤔  ...

Reply:
Pretty clear!